grudge/discretization.py

```diff
@@ -80,7 +80,10 @@ class DiscretizationCollection:
 
     def __init__(self, array_context: ArrayContext, mesh: Mesh,
                  order=None,
-                 discr_tag_to_group_factory=None, mpi_communicator=None,
+                 discr_tag_to_group_factory=None,
+                 volume_discr=None,
```

Comment:
"*The* volume discretization" will be less of a thing once we support multi-physics. So maybe prepare for that by using a dictionary from identifiers to discretizations.

Reply:
Good point, I've pushed some changes that remove all references to `volume_discr`